utils/scripts/prepare-for-multiyear.jl

```diff
@@ -0,0 +1,125 @@
+using DataFrames
+
+include("../csv-modifications.jl")
+
+# Local cleanup to ensure clean files
+run(`git restore test/inputs/ benchmark/EU/`)
+
+# Add year and lifetime to assets data
+apply_to_files_named("assets-data.csv") do path
+    change_file(path) do tcsv
+        add_column(tcsv, "year", 2030; position = 5)
+        add_column(tcsv, "lifetime", ""; position = 6)
```

Comment:
The parameter `lifetime` will be needed later, but it would be needed in the form of two parameters `economic lifetime` and `technical lifetime`. So I would suggest to remove this for now, otherwise we'll need to rename it, which is also fine if that's easy given the new script.

Reply:
I can remove and you add when you make the model changes, does that make sense?

Reply:
That sounds great. So I can add all the needed parameters later.